backends/xnnpack/test/ops/test_conv_transpose2d.py

```diff
@@ -0,0 +1,430 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import itertools
+import unittest
+from typing import Optional
+
+import executorch.extension.pybindings.portable_lib  # noqa[F401]
+import executorch.kernels.quantized  # noqa[F401]
```

Comment:
When doing local testing I am running into `ModuleNotFoundError: No module named 'executorch.kernels'`. I guess we need this to run the quantized ops when not lowered to XNNPACK. Can we skip running in the case of quantized graph and stop at serializing, perhaps that will eliminate the need to include this module?